LoopKitUI/Views/Settings Editors/InsulinModelSelection.swift

```diff
@@ -99,6 +115,22 @@ public struct InsulinModelSelection: View, HorizontalSizeClassOverride {
         }
     }
     
+    private var contentWithSaveButton: some View {
+        VStack {
+            content
+            Button(action: { self.onSave?(self.viewModel.insulinModelSettings) }) {
+                Text(mode.buttonText)
+                    .actionButtonStyle(.primary)
```

Comment:
Ah, I forgot to mark this extension method as deprercated when I rewrote ActionButtonStyle to use the ButtonStyle protocol. Can you make this `.buttonStyle(ActionButtonStyle(.primary))` instead?